conf/ocsci/rosa.yaml

```diff
@@ -0,0 +1,34 @@
+---
+ENV_DATA:
+  platform: 'rosa'
+  deployment_type: 'managed'
+  addon_name: 'ocs-converged'
+  configs: 'aws,stage'
+  worker_instance_type: 'm5.2xlarge'
+  worker_replicas: 3
+  master_replicas: 3
+  # size of storage cluster, default is 1 TiB
+  size: '1'
+  # notification_email_0: MAIL USED FOR SENDGRID
```

Comment:
If it's related to reporting I don't think it belongs to env data as it's not related anyhow with environment. I think it should be part of REPORTING section.

Also I would make the structure like:
```yaml
REPORTING:
  rosa_notification_email:
    - mail@address1.com
    - mail@address2.com
    - mail@address3.com
```

Not sure if there is some limitation of emails like max 3 or it can be more?

Reply:
There are 3 parameters for the addon. I will move it to REPORTING section.